packages/roosterjs-editor-types/lib/interface/ImageEditOptions.ts

```diff
@@ -1,4 +1,5 @@
 import ModeIndependentColor from './ModeIndependentColor';
+import { ImageEditOperation } from '../enum';
```

Comment:
Please fix this import, And also import CompatibleImageEditOperation using **type import**:

```ts
import { ImageEditOperation } from '../enum/ImageEditOperation';
import type { CompatibleImageEditOperation } from '../compatibleEnum/ImageEditOperation';
```

Reply:
Fixed